keyboards/mechlovin/zed60/zed60.h

```diff
@@ -64,3 +64,16 @@
     { K40, K41, K42, XXX, XXX, XXX, K46, XXX, XXX, XXX, K4A, K4B, K4C, K4D }, \
 }
 
+#define LAYOUT_60_iso( \
```

Comment:
Looks like you've defined the matrix in info.json (`"matrix": [*, *]`)? No need for it to be defined here.

Reply:
Thank you for the suggestion. 
Undefined!